wcivf/apps/elections/templates/elections/election_view.html

```diff
@@ -46,9 +45,11 @@ <h3>{{ object.nice_election_name }}</h3>
                         {% endif %}
 
                     {% else %}
-                        {% if not object.in_past and not election.slug == 'parl.2017-06-08' %}
-                            <p><a href="{{ object.ynr_link }}">{% trans "Add some candidates at our candidate crowd-sourcing site" %}</a></p>
-                        {% endif %}
+                        {% for postelection in object.postelection_set.all %}
+                            {% if not postelection.past_expected_sopn_day and not election.slug == 'parl.2017-06-08' %}
+                                <p><a href="{{ object.ynr_link }}">{% trans "Add some candidates at our candidate crowd-sourcing site" %}</a></p>
```

Comment:
We can drop the parl 2017 logic I think, and just stick with the first conditional.